src/Common/Core/Impl/Idle/IdleTimeAction.cs

```diff
@@ -4,10 +4,8 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.Common.Core.Shell;
-using Microsoft.Languages.Core.Utility;
-using Microsoft.Languages.Editor.Shell;
 
-namespace Microsoft.Languages.Editor.Tasks {
+namespace Microsoft.Common.Core.Idle {
```

Comment:
It may actually come useful in the broker for updating host load which is currently on timer. the idle does not rely necessarily on main thread activity, it is tied to 'idle service' which can be implemented as desired - such as timer.